apps/tlon-mobile/src/screens/Onboarding/ReserveShipScreen.tsx

```diff
@@ -32,10 +32,11 @@ export const ReserveShipScreen = ({ navigation }: Props) => {
   );
 
   useEffect(() => {
-    if (!signupContext.didCompleteSignup) {
-      signupContext.setDidCompleteSignup(true);
+    if (!signupContext.didCompleteOnboarding) {
+      signupContext.setOnboardingValues({ didCompleteOnboarding: true });
     }
-  }, [signupContext]);
+    signupContext.kickOffBootSequence();
+  }, []);
```

Comment:
Should this be empty?

Reply:
Yeah, we only want it to fire once when the screen mounts